languages/Assembler.groovy

```diff
@@ -262,25 +262,20 @@ def createAssemblerCommand(String buildFile, LogicalFile logicalFile, String mem
 	// add DD statements to the compile command
 	String assembler_srcPDS = props.getFileProperty('assembler_srcPDS', buildFile)
 
-	// Pass different allocations
-	// Case: BATCH - allocation SYSIN
-	if (!buildUtils.isCICS(logicalFile) && !buildUtils.isSQL(logicalFile)) assembler.dd(new DDStatement().name("SYSIN").dsn("${assembler_srcPDS}($member)").options('shr'))
-	//	else assembler.dd(new DDStatement().name("SYSCIN").ddref("SYSIN"))
-
-	// Case: CICS - translator overwrite Ddnames
-	if (buildUtils.isCICS(logicalFile)) assembler.setDdnames("SYSLIN,,,SYSLIB,SYSPUNCH,,,,,,,,,,,,,,")
-	else if (buildUtils.isSQL(logicalFile)) assembler.setDdnames("SYSLIN,,,SYSLIB,SYSCIN,,,,,,,,,,,,,,")
+	// Pass different input allocations
+	if (buildUtils.isCICS(logicalFile)) {  	// Case: CICS - translator overwrite Ddnames
```

Comment:
@dennis-behm  looks like Assembler.groovy is still using DD names as part of its solution

Reply:
Hi @drbruce-git Yes, this is correct. However, this is not part of the **linkEdit step** involving the MQ implementation. 

Because, the Assembler compiler does not have a  CICS or Db2 co-processor, it requires to precompile those files.
Rather than creating a new set of datasets to pass the temporary file, I would prefer to stay with the current implementation for the moment. But certainly we can address it via a different PR later on.